xml/admin_upgrade.xml

```diff
@@ -259,4 +259,213 @@ value: value)}'</command></screen>
+    that all your nodes are running on the latest v2 updates. You can check the
+    <filename>SUSEConnect</filename> package version to see if you are up to date.
+    To do so you will run a <command>salt</command> command to display the
+    package version installed on each node.
+   </para>
+   
+   <screen>
+&prompt.user;<command>docker exec -i  $(docker ps | grep salt-master | awk '{print $1}') \
+salt --batch 10 -P "roles:(admin|kube-(master|minion))" \
+cmd.run "rpm -q SUSEConnect"</command>
+
+Executing run on ['12cda3c374144d74804298bdee4d686c',
+                  '9b6d8d28393045c0914c959d0a5c0e33',
+                  '73b92dd7816147058c3d0fbb67fb18f9',
+                  'admin']
+admin:
+    SUSEConnect-0.3.11-3.15.1.x86_64
+jid:
+    20180809103558881056
+retcode:
+    0
+73b92dd7816147058c3d0fbb67fb18f9:
+    SUSEConnect-0.3.11-3.15.1.x86_64
+jid:
+    20180809103558881056
+retcode:
+    0
+9b6d8d28393045c0914c959d0a5c0e33:
+    SUSEConnect-0.3.11-3.15.1.x86_64
+jid:
+    20180809103558881056
+retcode:
+    0
+12cda3c374144d74804298bdee4d686c:
+    SUSEConnect-0.3.11-3.15.1.x86_64
+jid:
+    20180809103558881056
+retcode:
+    0
+   </screen>
+   
+   <para>
+    If the package version is <literal>0.3.11-3.15.1</literal> (or higher) you
+    have the latest updates from the v2 channel installed.
+   </para>
+   </sect2>
+   
+   <sect2 xml:id="sec.deploy.upgrade.caasp2.timer">
+    <title>Disable Automatic <literal>transactional-update</literal></title>
+    <para>
+    To begin with the upgrade procedure, you first must disable the automatic
+    transactional update mechanism to avoid conflicts. To do so you must run a
+    <command>salt</command> across the nodes to disable the 
+    <literal>transactional-update.timer</literal>.
+    </para>
+    <screen>
+&prompt.user;<command>docker exec -i $(docker ps | grep salt-master | awk '{print $1}') \
+salt --batch 10 -P "roles:(admin|kube-(master|minion))" \
+cmd.run "systemctl disable --now transactional-update.timer"</command>
+
+Executing run on ['5f6688bbeac94d2ab5c4330dc7043fb2',
+                  'c3afd049edbe43afb4e2e5913a88291b',
+                  '5bf346291a18406290886c2e2f7c3e3f',
+                  'admin']
+
+5bf346291a18406290886c2e2f7c3e3f:
+    Removed symlink /etc/systemd/system/timers.target.wants/transactional-update.timer.
+jid:
+    20180807122220543037
+retcode:
+    0
+admin:
+    Removed symlink /etc/systemd/system/timers.target.wants/transactional-update.timer.
+jid:
+    20180807122220543037
+retcode:
+    0
+c3afd049edbe43afb4e2e5913a88291b:
+    Removed symlink /etc/systemd/system/timers.target.wants/transactional-update.timer.
+jid:
+    20180807122220543037
+retcode:
+    0
+5f6688bbeac94d2ab5c4330dc7043fb2:
+    Removed symlink /etc/systemd/system/timers.target.wants/transactional-update.timer.
+jid:
+    20180807122220543037
+retcode:
+    0
+    </screen>
+   </sect2>
+   
```

Comment:
Add here something as extra step

"If you use SMT or SUSE Manager for mirroring channels: Mirror the  v3 channels on your server."